resources/chefdk/msi/assets/start-chefdk.ps1

```diff
@@ -0,0 +1,22 @@
+$conemulocation = "$env:programfiles\ConEmu\Conemu64.exe"
```

Comment:
Is it guaranteed to be in the PATH? (And does it need to be?)

Also, the 32-bit version of conemu is conemu.exe I presume?